modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/list/List.tsx

```diff
@@ -70,18 +191,19 @@ export default function List(props: IFDSViewSectionProps) {
 				</ClayTable.Head>
 
 				<ClayTable.Body>
-					{LIST_VISUALIZATION_MODE_FIELDS.map((field) => (
-						<ListField
-							field={field}
-							key={field.fieldId}
+					{listSections.map((listSection) => (
+						<ListSection
+							key={listSection.name}
+							listSection={listSection}
 							modalProps={props}
-							onSave={(fdsField) =>
-								setFieldValues({
-									...fieldValues,
-									[field.fieldId]: fdsField,
-								})
-							}
-							value={fieldValues[field.fieldId]}
+							onSelect={({closeModal, selectedField}) => {
```

Comment:
So now it is onSelect instead of onSave
![](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExOXBnamhtZzNzcGt6MXQzNm50YWcweXQ3Y2UzY21oc2UxeHZkMXBzNCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/flWn1Cyk9HcVIj9xWy/giphy.gif)

Reply:
Yep. The `Save` button no longer does the saving, only returns selected field, so I had no choice :smile: